e2e/etcd_test.go

```diff
@@ -424,6 +424,9 @@ func (epc *etcdProcessCluster) StopAll() (err error) {
 func (epc *etcdProcessCluster) Close() error {
 	err := epc.StopAll()
 	for _, p := range epc.procs {
+		if p == nil { // newEtcdProcess failed in the middle
```

Comment:
Just added more comment, and changed commit body to not close that issue. PTAL. Thanks.